moviesurfer/cli/main.py

```diff
@@ -1,2 +1,28 @@
-def main():
-    print("Hello from CLI, functionality is to be added")
+import logging
+
+import click
+
+from moviesurfer.data.download_data import download_data
+
+logger = logging.getLogger(__name__)
+
+
+@click.group(chain=True)
+def cli():
+    logger.info("moviesurfer cli started")
+
+
+@cli.command()
+@click.argument("download")
+def data(download):
```

Comment:
data may have multiple subcommands too
```suggestion
@click.group()
def data():
    pass
    
    
@data.command()
def download():
    download_data()
```

Reply:
i don't understand, woud you prefer to switch from arguments or subcommands? may i see some examples?

Reply:
Sure, here is why. `download` is not the only sub-command which we will have. I we want to add more, then this can quickly become messy. e.g.
```python
@cli.command()
@click.argument("download")
@click.argument("preprocess")
@click.argument("split")
@click.argument("cross-val")
def data(download, preprocess, split, cross-val):
    # now you need to check which one is not None
    # and be sure that only one is not none
```
+ each one may have arguments

The second pro in favor of sub-command is that it is a function, not an argument. It is a verb (to download). How do you distinguish that against usual arguments like `output-folder`?